lib/__tests__/_cookieUtils.test.ts

```diff
@@ -69,10 +81,12 @@ describe("cookieUtils", () => {
       AlgoliaInsights.setUserToken("007");
     });
     it("should return the current userToken", () => {
+      AlgoliaInsights.setUserToken("007");
```

Comment:
We already have a `beforeEach` function that calls the function on the line 69.

Reply:
https://github.com/algolia/search-insights.js/pull/141/commits/c73dff6202aa65d18dd9e1bc28577c02a9e7d888